src/VisualStudio/LiveShare/Impl/Shims/FindReferencesHandlerShim.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.LanguageServer.Handler;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Microsoft.VisualStudio.LiveShare.LanguageServices;
+
+namespace Microsoft.VisualStudio.LanguageServices.LiveShare
+{
+    [ExportLspRequestHandler(LiveShareConstants.TypeScriptContractName, Methods.TextDocumentReferencesName)]
+    internal class TypeScriptFindReferencesHandlerShim : AbstractLiveShareHandlerOnMainThreadShim<ReferenceParams, object[]>
```

Comment:
So is find references all that's left now for TypeScript or there's other stuff still using it?

Reply:
other stuff.  Will make that clear in a followup PR that deletes the rest of the roslyn server stuff (except for TS)